crates/driver/src/domain/competition/mod.rs

```diff
@@ -302,6 +337,70 @@ impl Competition {
         auction_id: Option<i64>,
         solution_id: u64,
         submission_deadline: u64,
+    ) -> Result<Settled, Error> {
+        let (response_tx, response_rx) = oneshot::channel();
+
+        let request = SettleRequest {
+            auction_id,
+            solution_id,
+            submission_deadline,
+            response_sender: response_tx,
+        };
+
+        self.settle_queue.try_send(request).map_err(|err| {
+            tracing::error!(?err, "Failed to enqueue /settle request");
```

Comment:
nit: Maybe this could be a warning and we handle a submission error with some metric on the autopilot side, to avoid spamming the alert channel.